AccountService.js

```diff
@@ -40,4 +40,71 @@ export class AccountService {
     });
     return response.data;
   }
+
+  /**
+   * @method accounts
+   * @param {Object} config
+   * @param {string} config.url - the base url
+   * @param {string} config.email - the user's email
+   * @param {string} config.after - an account's DID
+   * @param {integer} config.limit - how many accounts to return
+   * @return {Array} data
+   * @description calls on the index route and returns all
+   * accounts that match the email passed in.
+   */
+  async accounts({url = this.config.urls.base, email, after, limit}) {
```

Comment:
This naming convention (`accounts`) doesn't match. It should be `get`, I think. Others?

Reply:
yup, making it correspond to the backend API makes sense, `get` `getAll` that sort of thing.

Reply:
no actually get weirdly corresponds to get /:account
exists is the route this shares
so I would need to remove exists somehow.
maybe just rename this to getAccounts?